src/excmds.ts

```diff
@@ -2567,6 +2567,38 @@ export async function viewcontainers() {
             .replace(/ /g, "%20")
 }
 
+/** Opens the current tab in another container.
+
+    This is probably not a good idea if you care about tracking protection!
+    Transfering URLs from one container to another allows websites to track
+    you across those containers.
+
+    Read more here:
+    https://github.com/mozilla/multi-account-containers/wiki/Moving-between-containers
+
+    @param containerName The container name, fuzzy matched like `-c` on [[tabopen]]. Leave empty to uncontain.
+ */
+//#background
+export async function recontain(containerName = "") {
```

Comment:
I think we should probably call this function `containermove` to match up with the other container management commands. You can put an alias in the default config for `recontain` if you like.